pkg/http/api.go

```diff
@@ -35,6 +35,7 @@ import (
 
 // API returns a list of HTTP endpoints for Dapr
 type API interface {
+	channel.SetAppChannel
 	APIEndpoints() []Endpoint
```

Comment:
We need to expose `SetActorRuntime(actors actors.Actors)` method because each API struct has actor runtime instance. however,  initActor is called after app is ready. so we need to call this method after initActor.